R/findC.R

```diff
@@ -9,7 +9,10 @@
 #' @param iter a numeric value with number of penalty
 #' @return a vector of length (10 or indicated by iter) different C to be tested
 #'
-#' @examples findC(c(0, 5, 10, 15, 20, 25), max.lag = 1, iter = 15)
+#' @examples
```

Comment:
The function description is:
> This function computes the best C using the timepoints and max lag in the dataset

Do we need to describe what it is calculating?  It isn't actually the "best" C, but rather the list of values to test.

Can you add an example that uses `pi`?

Do we need to check whether `pi` and `iter` are within the valid ranges here and in other functions?

Reply:
I modified the function description.  Please confirm it is correct.